### DISCUSSION TITLE
When I create a new codespace, It shows this. It wiil last a long time and nohting change.

### DISCUSSION THREAD

**Discussion by ZhengZHU1106** [community/community#33612] [Codespaces]
<img width="962" alt="image" src="https://user-images.githubusercontent.com/49674480/191328801-771e5224-ad9d-4cf3-9c0a-3184bd0f0406.png">

**Comment by bjbennet** [comment]
Hi @JuCheng1997, sorry to hear you are having this issue. Does the codespace ever finish setting up? Can you please send me the name of the codespace that is having this behavior so I can investigate? Thanks.

**Comment by ZhengZHU1106** [comment]
It works in my Windows computer with chrome, but before in the MacBook with
chrome open can not be used

Bryan Bennett ***@***.***> 于2022年9月20日周二 20:08写道：

**Comment by bjbennet** [comment]
Chrome on Mac should be full supported. I'd be happy to investigate further if you have a codespace name you can provide.

**Comment by ZhengZHU1106** [comment]
Sure. Here you are.
JuCheng1997/Codespace_forPython (github.com)
<https://github.com/JuCheng1997/Codespace_forPython>

Bryan Bennett ***@***.***> 于2022年9月21日周三 01:20写道：
